scipy/signal/tests/test_signaltools.py

```diff
@@ -1295,7 +1295,7 @@ def test_odd_strides(self, xp):
         # us into wrong memory if used (but it does not need to be used)
         dummy = xp.arange(10, dtype=xp.float64)
         a = dummy[5:6]
-        a.strides = 16
+        a = xp.lib.stride_tricks.as_strided(a, strides=(16,))
```

Comment:
```suggestion
        a = np.lib.stride_tricks.as_strided(a, strides=(16,))
```

because otherwise (https://github.com/scipy/scipy/actions/runs/15788333110/job/44509424300?pr=23188), grumble grumble

```
>       a = xp.lib.stride_tricks.as_strided(a, strides=(16,))
            ^^^^^^
E       AttributeError: module 'scipy._lib.array_api_compat.numpy' has no attribute 'lib'
```

Reply:
gh-23199 now merged as so